src/main/resources/Development-documentation/gerrit-sceanrios.md

```diff
@@ -0,0 +1,224 @@
+# Gerrit scenarios
+
+These scenarios describes possible use cases that needs consideration during the
+implementation and update of the plugin
+
+<!-- TOC -->
+
+- [Gerrit scenarios](#gerrit-scenarios)
+  - [Create patch set - CASE1](#create-patch-set---case1)
+  - [Create patch set with rebase - CASE2](#create-patch-set-with-rebase---case2)
+  - [Create patch set with merge - CASE3](#create-patch-set-with-merge---case3)
+  - [Create patch set no previous events - CASE4](#create-patch-set-no-previous-events---case4)
+
+<!-- /TOC -->
+
+## Create patch set - CASE1
+
```

Comment:
Please add a stort explenation of each scenario before preconditions

Reply:
Added an explanation in 041b7cd